lib/main.dart

```diff
@@ -1,177 +1,537 @@
+                                .push(navigateToNewTodoItemPage());
+                          },
+                          maxLines: 2,
+                          style: const TextStyle(fontSize: 20),
+                          decoration: const InputDecoration(
+                              border: OutlineInputBorder(
+                                  borderRadius: BorderRadius.zero),
+                              hintText: 'Capture more things on your mind...'),
+                          textAlignVertical: TextAlignVertical.top),
+
+                      // On tablet and up
+                      tabletBody: TextField(
+                          key: textfieldKey,
+                          controller: TextEditingController(),
+                          keyboardType: TextInputType.none,
+                          onTap: () {
+                            Navigator.of(context)
+                                .push(navigateToNewTodoItemPage());
+                          },
+                          maxLines: 2,
+                          style: const TextStyle(fontSize: 30),
+                          decoration: const InputDecoration(
+                              border: OutlineInputBorder(
+                                  borderRadius: BorderRadius.zero),
+                              hintText: 'Capture more things on your mind...'),
+                          textAlignVertical: TextAlignVertical.top),
+                    ),
 
-        // Menu drawer (`endDrawer` comes from the right of the screen)
-        endDrawer: SizedBox(
-            width: MediaQuery.of(context).size.width * 1.0,
-            child: const Drawer(child: DrawerMenu())));
+                    // List of items
+                    Expanded(
+                      child: ListView(
+                        padding: const EdgeInsets.only(top: 40),
+                        scrollDirection: Axis.vertical,
+                        shrinkWrap: true,
+                        children: [
+                          if (items.isNotEmpty) const Divider(height: 0),
+                          for (var i = 0; i < items.length; i++) ...[
+                            if (i > 0) const Divider(height: 0),
+                            Padding(
+                              padding:
+                                  const EdgeInsets.symmetric(vertical: 8.0),
+                              child: ItemCard(item: items[i]),
+                            )
+                          ],
+                        ],
+                      ),
+                    ),
+                  ],
+                ),
+              );
+            }
+
+            // If the state of the TodoItemList is not loaded, we show error.ˆ
+            else {
+              return const Center(child: Text("Error loading items list."));
+            }
+          },
+        ));
   }
 }
 
-class MyTextField extends StatefulWidget {
-  const MyTextField({super.key});
+// PAGES ----------------------------
```

Comment:
this class seems to have to much responsibility. what about segregating different pieces in folders? an example:

https://github.com/dart-pacotes/.bricks/tree/master/aplicacao/__brick__/lib/presentation

Reply:
It is 😅 . In fact, everything is on `main.dart`.

I decided to keep it contained on this PR so it's easier to review (it's quite a big PR so I don't want to overload the reviewer).
Was hoping to get a proper refactor and concern-splitting on a different PR. 

I'll open an issue now and link your awesome [mason bricks](https://pub.dev/packages/mason_cli) repo 😄 .